.github/workflows/python-release.yml

```
@@ -66,13 +66,13 @@ jobs:
         with:
           python-version: ${{ env.PYTHON_VERSION }}
 
+      - name: Install uv
+        run: pip install uv
+
```

```suggestion
      - name: Install the latest version of uv
        uses: astral-sh/setup-uv@v4
        with:
          enable-cache: true
```

You don't need/want to be using `pip` for installations.